web/viewer.js

```diff
@@ -1615,11 +1620,14 @@ function webViewerInitialized() {
   document.getElementById('sidebarToggle').addEventListener('click',
     function() {
       this.classList.toggle('toggled');
+      var wasSidebarOpen = outerContainer.classList.contains('sidebarOpen');
+      PDFViewerApplication.sidebarOpen = !wasSidebarOpen;
+      if (PDFViewerApplication.sidebarOpen) {
+        sidebarContainer.classList.remove('hidden');
+      }
+      PDFViewerApplication.forceRendering();
```

Comment:
Sorry, I forgot to mention this previously: Can you also add a comment explaining why the code now calls `forceRendering` before opening the sidebar?
This way we reduce the risk that someone moves it back, thus causing a UX regression, in a future refactoring.